docs/tutorials/RemovingFields.md

```diff
@@ -100,7 +100,7 @@ In `src/test/data/`, data meant for testing purposes are stored. While keeping t
 
 ```json
 {
-  "anime": [ {
+  "animes": [ {
     "name": "Person with invalid name field: Ha!ns Mu@ster",
```

Comment:
shouldnt this part be changed too? name phone and email should probably be changed to.. name may stay as name but phone and email does not make sense here